Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -275,6 +293,29 @@ class AttachmentSaveEventListener final : public EventListener {
     WeakPtr<HTMLAttachmentElement, WeakPtrImplWithEventTargetData> m_attachment;
 };
 
+void HTMLAttachmentElement::updateProgress(const AtomString& progress)
+{
+    if (!m_progressElement)
+        return;
+
+    bool validProgress = false;
+    float value = progress.toFloat(&validProgress);
+    if (validProgress && std::isfinite(value)) {
+        StringBuilder style;
```

Comment:
I think we prefer to use `makeString` / `makeAtomString` when possible.

Reply:
Thanks for the tip, much better indeed!